scripts/fix-volumes.sh

```diff
@@ -13,26 +13,35 @@ set -o posix    # more strict failures in subshells
 IFS=$'\n\t'
 # ---- End unofficial bash strict mode boilerplate
 
+if [[ "$#" -ne 1 ]]; then
+  echo "Usage: $0 [desired_owner_user]" 1>&2
+  exit 1
+fi
+
 volumes=$(mount | grep -E '^/dev/' | grep -Ev ' on /etc/' || true)
 if [[ -z "${volumes}" ]]; then
+  echo "fix-volumes: No volumes found"
   exit
 fi
-owner=$(stat -c "%u:%g" .)
+
+desired_owner_user=$1
+owner="$(id -u ${desired_owner_user}):$(id -g ${desired_owner_user})"
```

Comment:
This part is the main change in this PR, how we determine the owner

Reply:
If I'm reading this diff right, we're now just hardcoding "node" which brings back the original mismatch issue on linux we had.